metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_config.py

```diff
@@ -119,13 +119,13 @@ class BigQueryV2Config(
         description="Generate usage statistic",
     )
 
-    capture_table_label_as_tag: bool = Field(
-        default=False,
+    capture_table_label_as_tag: AllowDenyPattern = Field(
+        default=AllowDenyPattern.allow_all(),
         description="Capture BigQuery table labels as DataHub tag",
     )
 
-    capture_dataset_label_as_tag: bool = Field(
-        default=False,
+    capture_dataset_label_as_tag: AllowDenyPattern = Field(
+        default=AllowDenyPattern.allow_all(),
```

Comment:
isn't this effectively changing the default from false to true?

I think it should be `Union[Literal[False], AllowDenyPattern]`, and the usages should check `if capture_dataset_label_as_tag and capture_dataset_label_as_tag.allowed(...)`

Reply:
Indeed, it was not clear to me how to keep the default to False. 
I have just committed the change, for use I am performing the following check:
`if tags and self.config.capture_dataset_label_as_tag:`
and for each label I check that the key matches what is indicated in the recipe.
Thank you very much for the quick reply!